gillespy2/solvers/numpy/CLE_solver.py

```diff
@@ -142,12 +141,16 @@ def run(self=None, model=None, t=20, number_of_trajectories=1, increment=None, s
             display.
         :type live_output_options: dict
 
-        :param timeout:
-        :param resume:
-        :param tau_tol:
-        :param kwargs:
+        :param timeout: If set, if simulation takes longer than timeout, will exit.
+        :type timeout: int
+        
+        :param resume: Result of a previously run simulation, to be resumed
+        
+        :param tau_tol: Tolerance level for Tau leaping algorithm.  Larger tolerance values will
+            result in larger tau steps. Default value is 0.03.
+        :type tau_tol: float
 
-        :returns:
+        :returns: a list of each trajectory simulated.
```

Comment:
solver run functions always return result objects not lists.  Also missing the rtype tag for the return type